### DISCUSSION TITLE
Please change the depressing brown picture at the start of the program

### DISCUSSION THREAD

**Discussion by andreysneg** [dbeaver/dbeaver#20554] [Q&A] [answered]
Please change the depressing brown ugly picture at the start of the program.

When I see her, I immediately do not want to work. Now I close eyes when start DBeaver .

Maybe programm have options for disable this picture ? 

Thank you

**Comment by E1izabeth** [comment, marked as answer]
You may try adding `nosplash` parameter.
See more here https://dbeaver.com/docs/wiki/Command-Line

**Comment by andreysneg** [comment]
Thank you. It works.

P.S. As a designer, I advise you to change the splach and abandon the brown light in the design of the program and the site. 
Brown color does not motivate and repels.

Use bright orange - #EB8A20  . And your sales go up 2 times

Example:
https://bit.ly/3PR3BB2

**Comment by E1izabeth** [comment]
Thank you for suggestion

**Comment by NotSignificant** [comment]
Please stop racism and please don't support the orange man. :D

Joking aside. I personally don't care about splash screens. And the dbeaver splash screen is one of those i can't see, because i'm using it as an eclipse plugin (of which the splash screen is blue and orange btw).

And also... The last thing, you should think about, are splash screen colors, especially if your software doesn't work well or doesn't work at all (what doesn't seem to be the case here).
